src/hooks/requests/useVendors.ts

```diff
@@ -62,3 +70,125 @@ export const useUpdateVendor = (vendorId: string) => {
     isLoading,
   }
 }
+
+export const useFetchSkills = () => {
+  const { isLoading, isError, data } = useQuery<GetSkillsPayload>({
```

Comment:
**[non-blocker]**

Not too important, but the type argument passed to useQuery should actually be the type of the response we get from BE. so basically the content of GetSkillsPayload type is correct, but the naming could use "Response" instead of "Payload"